ocs_ci/ocs/resources/pvc.py

```diff
@@ -74,6 +74,12 @@ def backed_pv_obj(self):
         pv_obj.reload()
         return pv_obj
 
+    def delete(self, wait=True, force=False, validate_pv_deleted=False):
+        result = super().delete(wait=wait, force=force)
+        if validate_pv_deleted:
+            self.backed_pv_obj.ocp.wait_for_delete(resource_name=self.backed_pv)
```

Comment:
This validation always fails since  backed_pv_obj method will reloads the pvc instance which is already deleted in line 78.

Reply:
Thanks, changed accordingly without verification due to PVC not getting bound bug.